lib/utils.js

```diff
@@ -31,6 +32,13 @@ function trimAllParents(paths, parents) {
   return paths.map(path => trimParents(path, parents))
 }
 
+function isValidExcludingGit(file) {
```

Comment:
Maybe this would be a little easier to understand what the function does?

```suggestion
function isFileValidAndNotGitRelated(file) {
```

Reply:
`isFileValidAndExcludesGit` or `isFileValidAndUnrelatedToGit` maybe?

Reply:
As we discussed today with @elrayle, naming this `isGitFile` or similar and using the result appropriately at the call site is probably the cleanest solution 👌🏼